scripts/assert_no_yarn_warnings.sh

```diff
@@ -1,14 +1,28 @@
 #!/bin/bash
 
-# run yarn install --check-files, capturing stderr
+# List of warnings to ignore (add new patterns here)
+IGNORE_WARNINGS=(
+  'engine "deno" appears to be invalid'
+  'engine "bun" appears to be invalid'
+)
```

Comment:
In modern packages it's common to have multiple engines and Yarn 1 is throwing warning for these. So we had to ignore these for now.